config/config.example.js

```diff
@@ -1,19 +1,23 @@
-import firebase from "firebase";
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+import { getStorage } from "firebase/storage";
+import { getDatabase } from "firebase/database";
 
-var config = {
+const firebaseConfig = {
   apiKey: "",
   authDomain: "",
   databaseURL: "",
   projectId: "",
   storageBucket: "",
   messagingSenderId: "",
+  appId: "",
+  measurementId: "",
 };
-firebase.initializeApp(config);
 
-var MAP_API_KEY = "";
+const app = initializeApp(firebaseConfig);
+const auth = getAuth(app);
+const storage = getStorage(app);
+const db = getDatabase(app);
+const MAP_API = "";
 
-export const f = firebase;
-export const database = firebase.database();
-export const auth = firebase.auth();
-export const storage = firebase.storage();
-export const MAP_API = MAP_API_KEY;
+export { app, auth, storage, db, MAP_API };
```

Comment:
I hope you can understand that with this change all the existing functionality of the app will be broken. My advice is to use compact mode and create a different issue to convert compact into modular method.

Reply:
Yes, this change might affect other fuctionality, so what i am doing now for this pr converted them back to compat mode and raise an another issue for converting into modular method.

Reply:
I have converted them back to compat method and tested the signin/signup with email&password, facebook everything is working fine on both client and firebase. Soon i will push all the changes.